frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -954,8 +957,13 @@ export function MaterializeVolumeAnnotationModal({
         jobName={"materialize_volume_annotation"}
         suggestedDatasetSuffix="with_merged_segmentation"
         chooseSegmentationLayer
+        isBoundingBoxConfigurable={includesProofreading}
```

Comment:
only proofread layers support configuring the bbox? would it be easy to always support it when exporting volume annotations? no prio though.